src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -830,6 +930,30 @@ inline void rpp_store48_u8pln3_to_i8pkd3(Rpp8s *dstPtr, __m128i *px)
     _mm_storeu_si128((__m128i *)(dstPtr + 36), _mm_sub_epi8(_mm_shuffle_epi8(_mm_unpackhi_epi8(pxDst[3], pxDst[1]), pxMaskRGBAtoRGB), xmm_pxConvertI8));    /* store [R13|G13|B13|R14|G14|B14|R15|G15|B15|R16|G16|B16|00|00|00|00] */
 }
 
+inline void rpp_store96_u8pln3_to_i8pkd3(Rpp8s *dstPtr, __m256i *px)
+{
+    __m256i pxDst[8];
+    __m256i pxZero = _mm256_setzero_si256();
```

Comment:
Instead of defining here we should already have a predefined constant pxZero perhaps somewhere in rpp_cpu_simd? I know pZero exists for float __m256

Reply:
Updated